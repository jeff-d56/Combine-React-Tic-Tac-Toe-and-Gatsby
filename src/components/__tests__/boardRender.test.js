import React from "react";
import Board from "../TicTacToe/board";
import renderer from "react-test-renderer"

var testSqaures = [null, null, null, null, null, null,null, null, null];

describe('Test if board renders correctly at start of game', () => {
	it("renders correctly", () => {
    testSqaures = [null, null, null, null, null, null,null, null, null];
	
    const tree = renderer.create(
	<Board
        squares={testSqaures}
        onClick={false}
    />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  
});



