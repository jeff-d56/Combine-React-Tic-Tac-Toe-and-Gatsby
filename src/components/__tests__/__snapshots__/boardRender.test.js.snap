// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if board renders correctly at start of game renders correctly 1`] = `
<div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
  </div>
</div>
`;
